src/app/pages/common/ix/components/ix-modal/ix-modal.interface.ts

```diff
@@ -0,0 +1,4 @@
+export interface IxModalData {
```

Comment:
Not used.

Reply:
Got added back due to the `master` merge. Will remove.